public/Lab-Files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+</head>
+<body>
+    <h1>Welcome to Erin's Music Blog</h1>
+    <p>It is one of the best feelings when you have a song stuck in your head all day
+    and then you get in the car and it is magically on the radio.<br>
+    It's like the universe is confirming that you were meant to think of that song. I love it.
+    </p>
+    <p><strong>Story time:</strong><br><blockquote>My boss at work is a bit of a joker. He likes to make puns and likes to mess with people.<br>
```

Comment:
Make sure you validate your code!
It is invalid to nest a block quote inside a paragraph tag. However you can have a paragraph tag inside a block quote, so I suggest you switch them in the future.